pkgs/development/ocaml-modules/hkdf/default.nix

```diff
@@ -4,15 +4,14 @@ buildDunePackage rec {
   pname = "hkdf";
   version = "1.0.4";
 
-  minimumOCamlVersion = "4.07";
+  minimalOCamlVersion = "4.08";
```

Comment:
Why this change ? source haven't changed. If it is related to `duneVersion = "3";` can you move this check to `buildDunePackage` so we don't forget to update other package ?

Reply:
It’s not related to the version of Dune.